### PR TITLE
Feature/groups

### PR DESCRIPTION
**Pull request by jairo-campos-JD**

### CODE REVIEW

File: src/classes/shex-form-model.ts [hunk 1]
```diff
@@ -275,22 +277,26 @@ export class ShexFormModel {
...
-            graph.addQuad(this.getSubjectNode(fieldTerm.id), namedNode(`${NS_UI}parts`), groupId)
-
-            this.walkShape(
-              valueExpr,
-              groupId,
-              `${path}/@${this.localName(te.valueExpr)}`,
-              namedNode,
-              literal,
-              blankNode
+            // graph.addQuad(this.getSubjectNode(fieldTerm.id), namedNode(`${NS_UI}parts`), groupId)
```
Comment (james-martin-jd) [inline src/classes/shex-form-model.ts]: Remove commented code

File: src/utils/form-ui.ts [hunk 2]
```diff
@@ -288,9 +288,9 @@ export async function convertFormModel(documentUri: any, documentPod: any) {
     },
     'ui:parts': { ...model }
   }
-  if (!existDocumentPod) {
+  /* if (!existDocumentPod) {
```
Comment (james-martin-jd) [inline src/utils/form-ui.ts]: Remove commented code

Comment (jairo-campos-JD) [reply src/utils/form-ui.ts]: done

File: src/classes/shex-form-model.ts (hunk 1 above)
Comment (jairo-campos-JD) [reply src/classes/shex-form-model.ts]: done

**Pull request merged**